### PR TITLE
Ports Aurora's animated areas alarms

### PR DESCRIPTION
**Pull request by quardbreak**
Also returns a atmosalarm alarm

### CODE REVIEW

**Comment by quardbreak** [comment]
@Lohikar You did these, would you like to review this?

File: code/game/area/areas.dm [hunk 1]
```diff
@@ -28,6 +28,10 @@
 		power_environ = 0
 	power_change()		// all machines set to current power level, also updates lighting icon
 
+	icon = 'icons/turf/areas.dmi'
+	icon_state = "white"
+	blend_mode = BLEND_MULTIPLY
```
Comment (afterthought2) [inline code/game/area/areas.dm:33]: Why is this set here, not in the `/area` definition? Why does it make icon updates outside of `update_icon`?

**Review by afterthought2 — COMMENTED** [review]
Needs playtest to see if it's tolerable.

File: code/game/area/areas.dm (hunk 1 above)
Comment (Lohikar) [reply code/game/area/areas.dm:33]: This is where they should be, per the reasons discussed on discord.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**